documentation/config_file_puppetserver.markdown

```diff
@@ -73,6 +73,19 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 * The `profiler` settings configure profiling:
 
     * `enabled`: If this is set to `true`, Puppet Server enables profiling for the Puppet Ruby code. The default is `false`.
+    
+* The `product` settings configure update checking and analytics data collection:
```

Comment:
`webserver` is already separated out into a `webserver.conf` file in the packages.  Only `jruby-puppet`, `http-client`, and `profiler` are in the puppetserver.conf file by default.  It would be a bit harder to separate the latter ones out from `puppetserver.conf` now because of compatibility / upgrade implications but that may have been a better way to have done it if we could do it over again.